DPPP/BDAAverager.cc

```diff
@@ -64,9 +64,13 @@ void BDAAverager::BaselineBuffer::Clear() {
   std::fill(uvw, uvw + 3, 0.0);
 }
 
-BDAAverager::BDAAverager(double bl_threshold_time, double bl_threshold_channel)
-    : bl_threshold_time_(bl_threshold_time),
-      bl_threshold_channel_(bl_threshold_channel),
+BDAAverager::BDAAverager(const DP3::ParameterSet& parset,
+                         const std::string& prefix)
+    : bl_threshold_time_(parset.getDouble(prefix + "timethresholdlength", 0.0)),
+      bl_threshold_channel_(
+          parset.getDouble(prefix + "freqthresholdlength", 0.0)),
+      max_interval_(parset.getDouble(prefix + "maxinterval", 3600.0)),
```

Comment:
Even though an hour is long, I think the cleaner solution would be to set the default at 0, which means 'do not enforce a maximum'.